.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,7 @@
 Please fill in the fields below to submit a pull request.  The more information
 that is provided, the better.
 
-**Fixes issue #**:
+**Fixes #<ISSUE NUMBER>**:
```

Comment:
```suggestion
**Fixes**: #<ISSUE NUMBER>
```

I'm not sure whether this is valid syntax for auto-closing issues, but the after the issue number does not make sense. Maybe remove it completely?

Reply:
I updated my pr with your suggestion.